patches/master_patch.patch

```diff
@@ -159,6 +159,19 @@ index 5c3210a3136ce1615ab8310e27f36375d314dbfa..e6d15636d71cb1aa42ce24c2f8cb7903
            "safe_browsing::mojom::SafeBrowsing",
            "translate::mojom::ContentTranslateDriver"
          ],
+diff --git a/chrome/browser/chrome_service.cc b/chrome/browser/chrome_service.cc
+index 3ea50d9bfd1cca4a78585be2066d1f203a92b5e4..f3bfb605482b45d06fc20c40856dd72e3b3f310a 100644
+--- a/chrome/browser/chrome_service.cc
++++ b/chrome/browser/chrome_service.cc
+@@ -31,8 +31,6 @@ ChromeService::ChromeService() {
+   registry_.AddInterface(
+       base::Bind(&chromeos::Launchable::Bind, base::Unretained(&launchable_)));
+ #endif
+-  registry_.AddInterface(
```

Comment:
we shouldn't patch things like this. Please either include the deps or stub if necessary

Reply:
Rebased here: #de45148daefa72b55f3787740e0f628d59fbfd0b